jax/_src/scipy/linalg.py

```diff
@@ -63,7 +63,7 @@ def cho_solve(c_and_lower, b, overwrite_b=False, check_finite=True):
 
 @partial(jit, static_argnames=('full_matrices', 'compute_uv'))
 def _svd(a, *, full_matrices, compute_uv):
-  a = np_linalg._promote_arg_dtypes(jnp.asarray(a))
+  a = _promote_dtypes_inexact(jnp.asarray(a))
```

Comment:
I think this is wrong. Curious how come it's not caught by the test suite.

Reply:
Thanks for checking on this - is that something you'd be willing to add?

Reply:
Fixed in #10387